cloud/openstack/os_volume.py

```diff
@@ -63,6 +63,12 @@
        - Volume snapshot id to create from
      required: false
      default: None
+   volume_src:
+     description:
+       - Volume source to create from
```

Comment:
Also, we've got three nouns for potential input. `image`, `snapshot_id`, and `volume_src`. That's three different nouns with differing identifiers (none, id, src). It may be too late to fix `image`, but it would be good if the other two matched.